avaframe/com3Hybrid/com3Hybrid.py

```diff
@@ -54,21 +65,26 @@ def maincom3Hybrid(cfgMain, cfgHybrid):
     iterate = True
     resultsHybrid = {}
     while iteration < nIterMax and iterate:
-        # update the com1DFA mu value
+        # update the com1DFA mu value in configuration file
         updater = ConfigUpdater()
-        updater.read(hybridModelDFACfg)
+        updater.read(com1DFACfgFile)
         updater['GENERAL']['mu'].value = ('%.4f' % muArray[-1])
         updater.update_file()
 
         # Run dense flow with coulomb friction
-        dem, _, _, simDF = com1DFA.com1DFAMain(avalancheDir, cfgMain, cfgFile=hybridModelDFACfg)
+        dem, _, _, simDF = com1DFA.com1DFAMain(avalancheDir, cfgMain, cfgFile=com1DFACfgFile)
         simID = simDF.index[0]
         particlesList, timeStepInfo = particleTools.readPartFromPickle(avalancheDir, simName=simID, flagAvaDir=True,
                                                                        comModule='com1DFA')
+
+        # fetch configuration for DFAPathGeneration
+        DFAPathGenerationCfg= cfgUtils.getModuleConfig(DFAPath, fileOverride='', modInfo=False, toPrint=False,
+            onlyDefault=cfgHybrid['DFAPathGeneration_override']['defaultConfig'])
+        DFAPathGenerationCfg, cfgHybrid = cfgHandling.applyCfgOverride(DFAPathGenerationCfg, cfgHybrid, DFAPath, addModValues=True)
```

Comment:
What about reading cfg and applying override at the top for everyone? I know it is not how I did it but it does not make sense to read them in the loop

Reply:
done